docs/userguide.md

```diff
@@ -43,8 +43,10 @@ custom image](development.md#test-your-changes).
 
 ### Create Clusters
 
-The quickest way to set up clusters for use with the federation-v2 control
-plane is to use [minikube](https://kubernetes.io/docs/getting-started-guides/minikube/).
+The federation v2 control plane can run on any Kubernetes clusters >= v1.11. Steps for running federation v2 on
+Google Kubernetes Engine [(GKE)](https://cloud.google.com/kubernetes-engine/) have been [documented](./gke.md). However,
```

Comment:
I'd like we move GKE related content after L80 with the content as follows:

If you want to run federation v2 on Google Kubernetes Engine [(GKE)](https://cloud.google.com/kubernetes-engine/), please refer to [GKE document](./gke.md) for detailed steps.

This can also easy to be extended in future, like if people want to add Azure, IBM Cloud etc, they can just add:

If you want to run ....., refer to .....